wrapper/src/main/java/software/amazon/jdbc/util/RdsUtils.java

```diff
@@ -367,13 +386,27 @@ public boolean isGreenInstance(final String host) {
     return !StringUtils.isNullOrEmpty(preparedHost) && BG_GREEN_HOST_PATTERN.matcher(preparedHost).matches();
   }
 
+  public boolean isOldInstance(final String host) {
+    final String preparedHost = getPreparedHost(host);
+    return !StringUtils.isNullOrEmpty(preparedHost) && BG_OLD_HOST_PATTERN.matcher(preparedHost).matches();
+  }
+
+  public boolean isNoPrefixInstance(final String host) {
```

Comment:
I find this method name a bit cryptic since it does not specify what prefix we are checking for and it doesn't mention this is about blue/green nodes. Can we invert the implementation/usages and rename to `isGreenDns`